.github/workflows/template_techdocs.yml

```diff
@@ -6,19 +6,19 @@ on:
       publisher-type:
         required: false
         type: string
-        default: 'azureBlobStorage'
+        default: "azureBlobStorage"
```

Comment:
Is there any cause to use the double quotes? 🤔

Reply:
My linter forces me to do this 🤷

Reply:
Okay if we do it, I would prefer to do it for all configurations. 🙌 Otherwise I would revert it to have a unique style.

Reply:
Why did you ignore this message? 🤔